datatypes/papas/PParticle.h

```diff
@@ -0,0 +1,66 @@
+//
+//  Created by Alice Robson on 29/11/15.
+//
+//
+#ifndef PARTICLE_H
+#define PARTICLE_H
+
+#include "Definitions.h"
+#include "TLorentzVector.h"
+#include "TVector3.h"
+
+#include "PDebug.h"
+
+namespace papas {
+/// Main Particle class
+///
+/// Contains 4-momentum vector, particle id and accessor functions
+/// Intended to be used as the base for derived Particle classes
+
+class Particle {
+public:
+  Particle();
+  // Particle& operator=(const Particle& P) = default;
+  Particle(int pdgid, double charge, TLorentzVector tlv, double status = 1);
+  Particle(int pdgid, double charge);
+  Particle(IdType id, int pdgid, double charge, TLorentzVector tlv, double status = 1);
+  Particle(IdType id, int pdgid, double charge);
+  ~Particle() = default;
+  Particle(Particle&&) = default;
+  Particle& operator=(const Particle&) = default;  // {std::cout << "= Particle"; };
+  Particle(Particle& pfblock) = default;           //{std::cout << "COPY Particle";};
```

Comment:
the parameter name is misleading, as pfblocks are no Particles. Replace by "other", or "particle"